--- ref
+++ di
@@ -13,7 +13,7 @@
     Supported color formats: RGB 4:4:4, YCrCb 4:4:4
     Default (sRGB) color space is primary color space
     First detailed timing includes the native pixel format and preferred refresh rate
-    Display supports continuous frequencies
+    Display is continuous frequency
   Color Characteristics:
     Red  : 0.6396, 0.3300
     Green: 0.2998, 0.5996
@@ -45,7 +45,7 @@
                  Hfront  176 Hsync  88 Hback  296 Hpol P
                  Vfront    8 Vsync  10 Vback   72 Vpol P
     Display Range Limits:
-      Monitor ranges (Range Limits Only): 24-85 Hz V, 24-135 kHz H, max dotclock 600 MHz
+      Monitor ranges (Bare Limits): 24-85 Hz V, 24-135 kHz H, max dotclock 600 MHz
     Display Product Name: 'hdmi-1080p'
     CVT 3 Byte Timing Codes:
       CVT:  1920x1080   59.962844 Hz  16:9     67.158 kHz    173.000000 MHz
